enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+---
+title: multicluster-logs-traces-forwarding
+authors:
+  - "@periklis"
+  - "@JoaoBraveCoding"
+  - "@pavolloffay"
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+  - https://issues.redhat.com/browse/OBSDA-489
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Logs and Traces Forwarding
```

Comment:
The tracing team has concerns with moving forward with this proposal.

Our requirement is to provide OpenTelemetry features in multi-cluster environments which was facilitated by PR - https://github.com/rhobs/multicluster-observability-addon/pull/39#issuecomment-2065235658 and caused confusion about the support responsibilities and was halted for a long time without any proposal how to move it forward.

The observability organization is split into separate teams that develop, maintain, and support a set of operators and CRDs that provide in-cluster user-facing APIs. This proposal enables the distribution and reconciliation of individual CRDs (e.g. `ClusterLogForwarder`) to spoke clusters. Users have to understand and use these CRDs in both in-cluster and multi-cluster setups.

The proposal is to change `mcoa.openshift.io/signal={logging,tracing}` to `mcoa.openshift.io/kind={ClusterLogForwarder,OpenTelemetryCollector}` and structure the addon around kinds/CRDs rather than signals. This makes it straightforward which CRDs users interact with, support questions, and extensibility for the future.

In the arch calls we talked as well about integrating cluster observability operator (COO) into the addon for providing store capabilities in the spokes, which is a cross-signal operator, hence it fits nicely with the kind/CRD structure.

Reply:
Solving this as since this comment was made progress has been made in https://docs.google.com/document/d/1Yha0IvwmTOzc3HMqG0gj9edRjlPIzIAwvotIFqCBJ9E to ensure MCOA fits all the use cases